File: MPS/Models/OutboundOrder.cs
using System;
using System.Collections.Generic;

namespace MPS.Models
{
    public partial class OutboundOrder
    {
        public string Batch_id { get; set; }
        public string Pallet_id { get; set; }
        public int Order_pallet_count { get; set; }
        public DateTimeOffset? Expected_activation_time { get; set; }
        public string Status { get; set; }
        public int Index { get; set; }
        public string Source { get; set; }
        public int? Wms_link_id { get; set; }
        public int? Wms_request_status_read { get; set; }
        public int? Wms_output_status_read { get; set; }
        public DateTime? Automated_activation_time { get; set; }
        public long? Target { get; set; }
    }
}

File: MPS/ClientApp/src/reduxStore/outboundOrder.ts
// Variable Type
export interface OutboundOrder {
    batch_id: string | null;
    pallet_id: string;
    order_pallet_count: number;
    expected_activation_time: Date | null; // as in service will run as NOW();
    status: string;
    index: number;
    source: string | null;
    wms_link_id: string | null;
    wms_request_status_read: string | null;
    wms_output_status_read: string | null;
    automated_activation_time: Date | null;
    target: number; 
}